### PR TITLE
Update 01-login.md

### PR DESCRIPTION
**Pull request by joshcanhelp**
From the QS feedback channel

### CODE REVIEW

File: articles/quickstart/webapp/python/01-login.md [hunk 1]
```diff
@@ -104,14 +108,14 @@ def callback_handling():
 
 ## Trigger Authentication
 
-Add a `/login` route that uses the `Authlib` client instance to redirect the user to the [login page](/hosted-pages/login).
+Add a `/login` route that uses the `Authlib` client instance to redirect the user to the [login page](/hosted-pages/login). Replace `YOUR_CALLBACK_URL` in the snippet below with the **Allowed Callback URL** configured for your Application in Auth0.
```
Comment (damieng) [inline articles/quickstart/webapp/python/01-login.md]: "In Auth0" is a bit of unusual phrasing.  Maybe "for your Application in the Auth0 dashboard?" or something?

File: articles/quickstart/webapp/python/01-login.md [hunk 2]
```diff
@@ -104,14 +108,14 @@ def callback_handling():
...
 
 ```python
 # /server.py
 
 @app.route('/login')
 def login():
-    return auth0.authorize_redirect(redirect_uri='YOUR_CALLBACK_URL', audience='https://${account.namespace}/userinfo')
```
Comment (lbalmaceda) [inline articles/quickstart/webapp/python/01-login.md:114]: was this removed from the sample as well?

File: articles/quickstart/webapp/python/01-login.md (hunk 1 above)
Comment (joshcanhelp) [reply articles/quickstart/webapp/python/01-login.md]:
```suggestion
Add a `/login` route that uses the `Authlib` client instance to redirect the user to the [login page](/hosted-pages/login). Replace `YOUR_CALLBACK_URL` in the snippet below with the **Allowed Callback URL** configured for your Application in the Auth0 dashboard.
```

File: articles/quickstart/webapp/python/01-login.md (hunk 2 above)
Comment (joshcanhelp) [reply articles/quickstart/webapp/python/01-login.md:114]: I'm not seeing anything that matches this ... though I do see an audience mentioning userinfo here:

https://github.com/auth0-samples/auth0-django-web-app/blob/f2f5466311589acd954cd31e474baba5754d887e/01-Login/webappexample/settings.py#L145

Comment (lbalmaceda) [reply articles/quickstart/webapp/python/01-login.md:114]: @joshcanhelp That link points to the "DJango" webapp. This is "Python" webapp.

The code is still referencing the userinfo audience. It [defines it here](https://github.com/auth0-samples/auth0-python-web-app/blob/master/01-Login/server.py#L30-L31) and [uses it here](https://github.com/auth0-samples/auth0-python-web-app/blob/master/01-Login/server.py#L93). I'm OK with removing this audience now if the ID token is going to be verified and the claims be returned to the user, as if it were a /userinfo request result. But please keep both sample and article in sync.

Comment (joshcanhelp) [reply articles/quickstart/webapp/python/01-login.md:114]: Second reference should probably stay. Here's a PR for the first one:

https://github.com/auth0-samples/auth0-python-web-app/pull/52

**Review by lbalmaceda — APPROVED** [review]

**Pull request merged**